### ISSUE TITLE
Bring back financius app

### ISSUE THREAD

**Issue by tojanjob** [mvarnagiris/financius#421]
Dear developer
Please bring back financius expance manager app to work on latest android platforms.it was the best app I used for my expanse managing 
Tojan job

**Comment by mvarnagiris** [comment]
Hehe, you still used it? What did you like about it?

**Comment by tojanjob** [comment]
I liked it's simplicity and ease of use.it had everything that I wanted
from an expance manager app.the sad part was that I am not able to use it
with new phones as it is not compatible. although a lot of expance manager
apps in the Play Store still I haven't find anything better than financius
app for my use.it will be great if the app comes back
Thanks
Tojan job

**Comment by mlapkin** [comment]
Have to agree with @tojanjob , Financius was simple but did things the right way.

**Comment by tojanjob** [comment]
Yes, I agree.

**Comment by AbubakarSiddique** [comment]
Yes I was also a core user too but since many years I have shifted to one alternative [Money Manager: Expense Tracker](https://play.google.com/store/apps/details?id=com.ktwapps.walletmanager), which works for me. It's quite simple like financius as well as is actively developed.

**Comment by petergasper** [comment]
I am still using some closed source fork available on the play store, just blocking network permissions of that app to get rid of adds.
maybe with the current state of AI even non-android developer can refactor this treasure :)

**Comment by Gae-IT** [comment]
I am happy with version 0.9.11 which continues to work on my pixel updated today to android 16. I don't use the currency exchange or the categories or the backup, I just need to know the balances of my accounts and cards, and Financius is perfect like this, I would only add the unlock with the fingerprint
